### ISSUE TITLE
Crash on loading saved file

### ISSUE THREAD

**Issue by gitkrax** [chummer5a/chummer5a#1101]
### Environment
Chummer Version: Both 5.183.0 and Nightly-v5.183.41 exhibit this behavior.
Environment: Ubuntu 16.04.
Runtime: Not sure, but I'm guessing: Mono JIT compiler version 4.2.1 (Debian 4.2.1.102+dfsg2-7ubuntu4)

### Expected behaviour
Succesfully load saved character.

### Actual behaviour
Crash on choosing character file.

I can see the following errors in terminal:

```
Could not set X locale modifiers
System.IO.FileNotFoundException: Could not load file or assembly 'System.Windows.Forms.DataVisualization, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' or one of its dependencies.
File name: 'System.Windows.Forms.DataVisualization, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
  at (wrapper remoting-invoke-with-check) Chummer.frmMain:LoadCharacter (string,bool,string,bool)
  at Chummer.frmCharacterRoster.treCharacterList_DoubleClick (System.Object sender, System.EventArgs e) <0x41afa770 + 0x0009b> in <filename unknown>:0 
  at System.Windows.Forms.Control.OnDoubleClick (System.EventArgs e) <0x41afa6e0 + 0x00065> in <filename unknown>:0 
  at System.Windows.Forms.TreeView.HandleClick (Int32 clicks, System.Windows.Forms.MouseEventArgs me) <0x41af9540 + 0x00087> in <filename unknown>:0 
  at System.Windows.Forms.Control.WmLButtonUp (System.Windows.Forms.Message& m) <0x41af93d0 + 0x00109> in <filename unknown>:0 
  at System.Windows.Forms.Control.WndProc (System.Windows.Forms.Message& m) <0x41a28470 + 0x00217> in <filename unknown>:0 
  at System.Windows.Forms.TreeView.WndProc (System.Windows.Forms.Message& m) <0x41ac7c90 + 0x000cb> in <filename unknown>:0 
  at System.Windows.Forms.Control+ControlWindowTarget.OnMessage (System.Windows.Forms.Message& m) <0x41a28440 + 0x00024> in <filename unknown>:0 
  at System.Windows.Forms.Control+ControlNativeWindow.WndProc (System.Windows.Forms.Message& m) <0x41a28400 + 0x00036> in <filename unknown>:0 
  at System.Windows.Forms.NativeWindow.WndProc (IntPtr hWnd, Msg msg, IntPtr wParam, IntPtr lParam) <0x41a26d40 + 0x0031c> in <filename unknown>:0 
```

The first line is added on program start, rest of it is printed on crash.

### Steps to reproduce the behaviour
Try to load a previously saved character file in the environment mentioned above. Does not occur on Windows.

### Link to affected character file
I can reproduce the behavior with any character file I tried, but here is a minimal example:
[rigger.chum5.zip](https://github.com/chummer5a/chummer5a/files/472156/rigger.chum5.zip)

**Comment by chummer5a** [comment]
That's... interesting. Does it load properly if you don't use the character roster?

**Comment by joha4270** [comment]
I'm around 90% sure its the history graphs for karma/nuyen history in
career mode that is causing this issue.

At least that's where i would start looking.

Working from that hypothesis, the problem would get a decent workaround by
moving the "History" tab (think, name from memory) to its own file and
loading it upon first click.
Would need to confirm that's the actual issue first thought. Probably won't
have time to check the next few days, but I'll try to create a special
build without that tab and upload it here to confirm said hypothesis.

**Comment by gitkrax** [comment]
I'm not sure I understood the question, but:
It crashes both when choosing a character from the character roster and when choosing a saved file from the file system.

**Comment by chummer5a** [comment]
Shouldn't be the expense history, that character's still in creation mode and the datavisualisation only exists in career.

**Comment by chummer5a** [comment]
Can you please try with a newer version of Mono? http://www.pokorra.de/tag/mono seems to work for me, but I couldn't get it to crash with the default version I had installed either so mileage may vary.

**Comment by gitkrax** [comment]
Sorry, but probably no. I have a stable Ubuntu LTS with the default mono installed, which is also needed by other programs. I'm not interested in seeing whether something breaks.

After this bug report I used the Windows version, which corrupted my character file shortly before I finished character creation, so it's not been a great experience. I'm probably sticking to pen and paper for future characters.
